### ISSUE TITLE
Security vulnerability

### ISSUE THREAD

**Issue by mamodom** [Microsoft/vscode-extension-vscode#126]
See https://github.com/dominictarr/event-stream/issues/116

```
mamodom@MAMO-T470 :  (master)~/dev/vscode-extension-vscode$ npm ls event-stream flatmap-stream
vscode@1.1.21 /home/mamodom/dev/vscode-extension-vscode
├─┬ gulp-remote-src-vscode@0.5.0
│ └── event-stream@3.3.4
├─┬ gulp-symdest@1.1.0
│ └── event-stream@3.3.4  deduped
├─┬ gulp-untar@0.0.7
│ └── event-stream@3.3.4  deduped
└─┬ gulp-vinyl-zip@2.1.0
  └── event-stream@3.3.4  deduped

```

There are pull requests pinning `event-stream` to 3.3.4

https://github.com/ddliu/gulp-remote-src/pull/18
https://github.com/joaomoreno/gulp-symdest/pull/2
https://github.com/jmerrifield/gulp-untar/pull/15
https://github.com/joaomoreno/gulp-vinyl-zip/pull/13

**Comment by roonyh** [comment]
`gulp-remote-src-vscode` is still not updated. I think its fork of `gulp-remote-src`? `gulp-remote-src` is now updated.
Please update it if possible.

**Comment by mamodom** [comment]
You're absolutely right @roonyh here's the fork that publishes to [`gulp-remote-src-vscode`](https://www.npmjs.com/package/gulp-remote-src-vscode) https://github.com/joaomoreno/gulp-remote-src

@joaomoreno should update the repo's repository field so thtat npm links to the correct package

**Comment by mamodom** [comment]
Here's the actual repository: https://github.com/joaomoreno/gulp-remote-src @joaomoreno  already pinned the version. I opened a pr updating its repository url so that it's clear where's the source code of that package

**Comment by bpasero** [comment]
See https://code.visualstudio.com/blogs/2018/11/26/event-stream

We have meanwhile published `vscode@1.1.22`. To ensure your extension is using this extension, run `npm i vscode`. That should also update the `package-lock.json`.
